src/components/ProgressButton.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="flex flex-col gap-2 items-center w-fit">
+    <PrimaryButton
+      v-bind="props"
```

Comment:
```suggestion
      :label="label"
```

Die Lösung übergibt mit v-bind="props" auch dotAmount und dotProgress an die PrimaryButton-Komponente, obwohl diese dort gar nicht verwendet werden.
Außerdem ist die Schreibweise etwas implizit, gerade wenn nur eine einzelne Prop (label) tatsächlich weitergegeben wird.

Hinweis: In Vue 3 (ab Version 3.2) kann man Props mit gleichnamigen Variablen direkt per Shortcut übergeben. Statt :label="label" reicht einfach :label.